README.md

```diff
@@ -2,13 +2,21 @@
     <img src="assets/eso-logo-large.png" width="30%" align="center" alt="external-secrets">
 </p>
 
+*For the **Platform Teams***
+*Who **need a secure method to get secrets from an external manager to K8s services***
+*The **External Secrets Operator***
+*is a **Secrets Management Integration Tool***
+*that **brings Secrets from your secrute environemnts into k8s Workloads***
```

Comment:
Typo: Secrute

Reply:
wow :D thanks!

Reply:
And "environments" too

Reply:
I copy pasted the card contents... with all the typos included :s sorry for that.